src/pages/help/support.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Support
+description: We want to help you apply the IBM Design Language both correctly and creatively. Learn about the approvals process and where you can find answers to specific questions.
+
+---
+
+import { PageDescription } from 'gatsby-theme-carbon';
+
+<PageDescription>
+
+We want to help you apply the IBM Design Language both correctly and creatively. Learn about the approvals process and where you can find answers to specific questions.
+
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Collaboration and approvals</AnchorLink>
+  <AnchorLink>Questions?</AnchorLink>
+  <AnchorLink>Enablement</AnchorLink>
+</AnchorLinks>
+
+## Collaboration and approvals
+
+We are counting on your creativity to help bring the IBM Design Language to life in everything designed for IBM. Please loop in a member of the Brand Experience Design team early in the creative process for all significant new projects, so that we can help you learn to express the IBM Design Language both correctly and creatively.
+
+To advise the team on new projects and coordinate their participation, please send a message to our Slack channel [#ibm-design-language](https://ibm-studios.slack.com/messages/C0U35LB2B), with a brief description of the project, the timeline and the names of the main IBM and agency contacts. If you are an agency partner, or do not have access to IBM’s Slack channels, please email these details to: [designlanguage@us.ibm.com]() with the subject: IDL Sponsor User Group request.
+
+## Questions
+
+Please take some time to explore the content on this website before reaching out to the IBM Design team. The site is very comprehensive and most of the guidelines and components are well documented. However, if you still have a specific question:
+
+
+<ol class="bx--list--ordered">
+<li class="bx--list__item">
+
+Please check whether there is an existing answer on our [FAQ page]()
+
+</li>
+<li class="bx--list__item">
```

Comment:
You also don't have to use html for this, you can just use tick marks or numbers for lists in MDX:

i.e. 
```md
- this is an unordered list item
- etc
- etc

1. this is an ordered list item
2. etc
3. etc
```

Reply:
I have used that before but they wanted the carbon list component.

Reply:
@greeshmareddyg the theme supports nested lists